index.html

```diff
@@ -45,6 +45,178 @@ <h1>
       <img src="images/Normal Button/Secondary/Disabled.png" alt="">
     </section>
   </section>
+  <section class="r_works">
+    <section class="title">
+      <p>My Recent Works</p>
+      <p class="s"></p>
+    </section>
+    <!-- <div>1</div> -->
+    <section class="works grid-container">
+        <section class="grid_1-item">
+          <!-- <img src="#" alt="" class="img_place_holder"> -->
+          <section class="txt">
+            <p class="project_title">Multi-Post Stories Gain+Glory</p>
+            <ul class="tag_ctn">
+              <li class="tag"><span class="tag_txt">Ruby on Rails</span></li>
+              <li class="tag"><span class="tag_txt">CSS</span></li>
+              <li class="tag"><span class="tag_txt">Javascript</span></li>
+              <li class="tag"><span class="tag_txt">HTML</span></li>
+            </ul>
+            <section class="action_2 centered">
+              <button class="see">
+                <span class="see_txt">See Project</span>
```

Comment:
- For every `see_txt` class of mobile version, I can see you have used `width: 98px` in CSS.
- ![image](https://user-images.githubusercontent.com/60083437/235779249-8b95ae04-33fd-4ffb-b710-283b13168058.png)
- For this reason I can see an error in the `text` of the `See Project` button. The text inside the button is divided in two lines that doesn't match with the given design guideline.
- ![image](https://user-images.githubusercontent.com/60083437/235779401-7ef8bc68-f381-40dd-8b58-87a65703135b.png).
- I recommend you to `remove` `width: 98px` and check if its work. I hope it will work.
- [ ] Kindly fix this error in the `See Project` button.

Reply:
You are right. Ok done